src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -16929,6 +16929,35 @@ class B2 : A<string>
                 Diagnostic(ErrorCode.ERR_ScopedMismatchInParameterOfOverrideOrImplementation, "F6").WithArguments("s").WithLocation(17, 32));
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/73384")]
+        public void Indexer_EscapeInGetter()
```

Comment:
Do we have similar tests for interfaces?

Reply:
No and we had a dupe of almost the same check which causes the error to get dropped. Fixing.